### PR TITLE
[FIX] remove kafka clients from distributions

### PR DESCRIPTION
**Pull request by raphaelauv**

### CODE REVIEW

**Comment by raphaelauv** [comment]
to install the connector in kafka connect is only necessary this ->
Screenshot from 2022-03-23 16-52-04


but with `implementation "org.apache.kafka:kafka-clients:$kafkaVersion"` it put kafka-client jar inside
Screenshot from 2022-03-23 16-51-52

File: build.gradle [hunk 1]
```diff
@@ -121,7 +121,7 @@ dependencies {
     compileOnly "com.github.spotbugs:spotbugs-annotations:$spotbugsAnnotationsVersion"
     compileOnly "com.google.code.findbugs:jsr305:$jsr305Version"
 
-    implementation "org.apache.kafka:kafka-clients:$kafkaVersion"
+    compileOnly "org.apache.kafka:kafka-clients:$kafkaVersion"
```
Comment (jjaakola-aiven) [inline build.gradle:124]: Group to compile only, just one line up.

**Review by jjaakola-aiven — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (raphaelauv) [reply build.gradle:124]: I don't understand your message

Comment (jjaakola-aiven) [reply build.gradle:124]: Move the line on up, having empty line between the compileOnly and implementation.

Comment (raphaelauv) [reply build.gradle:123]: done

**Comment by jjaakola-aiven** [comment]
@raphaelauv I gave poor instructions. The grouping should go here https://github.com/aiven/http-connector-for-apache-kafka/pull/85/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R118. To the first group of compile only libraries. I am sorry to ask to for one more change.

**Comment by raphaelauv** [comment]
You can commit on this PR

**Review by jjaakola-aiven — APPROVED** [review]

**Pull request merged**